fighthealthinsurance/common_view_logic.py

```
@@ -1646,3 +1649,105 @@ async def format_response(response: dict[str, str]) -> str:
             logger.debug(f"Yielding {i}")
             yield i
+
+
+class StripeWebhookHelper:
+    @staticmethod
+    def handle_checkout_session_completed(request, session):
+        try:
+            payment_type = session.metadata.get("payment_type")
+
+            if payment_type == "interested_professional_signup":
+                InterestedProfessional.objects.filter(
```

Ruff (0.8.2)

`InterestedProfessional` may be undefined, or defined from star imports

(F405)